draft-ietf-quic-transport.md

```diff
@@ -73,9 +73,11 @@ informative:
     title: "Structured Streams: A New Transport Abstraction"
     author:
       - ins: B. Ford
-    date: 2007-08
+    date: 2007-10
```

Comment:
Looking at the kramdown-rfc2629 code, it doesn't look like it yet supports the bibxml7 directory.  Here are the currently-supported references:

~~~
        "RFC" => "bibxml", "I-D" => "bibxml3", "W3C" => "bibxml4", "3GPP" => "bibxml5",
        "ANSI" => "bibxml2",
        "CCITT" => "bibxml2",
        "FIPS" => "bibxml2",
        "IANA" => "bibxml2",
        "IEEE" => "bibxml2",
        "ISO" => "bibxml2",
        "ITU" => "bibxml2",
        "NIST" => "bibxml2",
        "OASIS" => "bibxml2",
        "PKCS" => "bibxml2",
~~~